R/utils-json.R

```diff
@@ -278,10 +281,51 @@ subsetByMonth <- function(pts, onlyMonth) {
   }
   return(pts)
 }
-getYvals_approvals <- function(object, num_vals){
-  ylim <- ylim(object)$side.2[1]
-  yvals <- rep(ylim, num_vals)
-  return(yvals)
+
+ApprovalInfoY <- function(object, times, c) {
+  # Compute top and bottom vertical position of approval bars.
+  # 
+  # Args:
+  #   object: A gsplot object.
+  #   times: "times" argument of R rep() function.
+  #   c: A scaling ratio to adjust top or bottom of approval bar rectangle.
```

Comment:
you should convert this documentation to Roxygen and put it right above. This is used by R to create help files for the package

```
#' compute top and bottom vertical position of approval bars.
#' @param object a gsplot object
#' @param times "times" argument of R rep() function.
#' @param sc a scaling ratio to adjust top or bottom of approval bar rectangle.
```